src/flownet/config_parser/_config_parser.py

```diff
@@ -1734,6 +1734,26 @@ def parse_config(
             "interpolation option for relative permeability."
         )
 
+    if (
+        config.model_parameters.equil.scheme != "regions_from_sim"
+        and config.model_parameters.equil.scheme != "individual"
+        and config.model_parameters.equil.scheme != "global"
```

Comment:
Shorter, and more readable?

```suggestion
        config.model_parameters.equil.scheme not in ["regions_from_sim", "individual", "global"]:
```

Reply:
Surely!